### PR TITLE
code review

### PR DESCRIPTION
**Pull request by jpopelka**
Various changes as a result of code review.

### CODE REVIEW

**Comment by jpopelka** [comment]
Can somebody take a look ?

**Comment by jpopelka** [comment]
Testing instance of release-bot for releasing conu is running at https://open.paas.redhat.com/console/project/release-bot-test/overview

Please review anybody.

File: .s2i-dev/bin/usage [hunk 1]
```diff
@@ -4,7 +4,7 @@ cat <<EOF
 This is a S2I release-bot image:
 To use it, install S2I: https://github.com/openshift/source-to-image
 Sample invocation:
-s2i build <configuration_repository> release-bot release-bot-app
+s2i build <configuration_repository> usercont/release-bot:dev release-bot-app:dev
```
Comment (phracek) [inline .s2i-dev/bin/usage]: Instead of `usercont` use `user-cont`, right? In the whole PR.

File: Dockerfile [hunk 2]
```diff
@@ -13,15 +13,15 @@ LABEL summary="Automated releasing from GitHub repositories" \
       io.k8s.display-name="Release Bot" \
       io.openshift.tags="builder" \
       io.openshift.s2i.scripts-url="$STI_SCRIPTS_URL" \
-      usage="s2i build <SOURCE-REPOSITORY> koscicz/release-bot <APP-NAME>"
+      usage="s2i build <CONFIGURATION-REPOSITORY> usercont/release-bot <APP-NAME>"
 
-RUN dnf install -y python3 python2 python-pip fedpkg git krb5-workstation
+RUN dnf install -y python2-pip fedpkg git krb5-workstation
```
Comment (phracek) [inline Dockerfile]: Any reason for removing python3?

File: Makefile [hunk 3]
```diff
@@ -1,7 +1,23 @@
 .PHONY: test
 
+IMAGE_NAME := usercont/release-bot
```
Comment (phracek) [inline Makefile]: typo, it should be user-cont, right?

File: .s2i-dev/bin/usage (hunk 1 above)
Comment (phracek) [reply .s2i-dev/bin/usage]: Uppss, usercont is image name and not GitHub repository.

Comment (jpopelka) [reply .s2i-dev/bin/usage]: [usercont](https://hub.docker.com/u/usercont) is our organization on docker hub. The image is  [usercont/release-bot](https://hub.docker.com/r/usercont/release-bot)

File: Dockerfile (hunk 2 above)
Comment (jpopelka) [reply Dockerfile]: because it's installed by default on fedora:27
The commit messages explain my steps.

**Comment by TomasTomecek** [comment]
Can't wait to try this on my projects.

**Pull request merged**